plasma_framework/migrations/1041_setup_vault.js

```diff
@@ -41,44 +41,54 @@ const setDepositVerifier = async (whatLog, ethVault, ethDepositVerifier, gnosisM
     const setDepositVerifierCall = web3.eth.abi.encodeFunctionCall(ethVault.abi.find(o => o.name === 'setDepositVerifier'), [ethDepositVerifier.address]);
     const gnosisSetDepositVerifier = web3.eth.abi.encodeFunctionCall(gnosisMultisigAbi, [ethVault.address, 0, setDepositVerifierCall]);
     const gas = await web3.eth.estimateGas({ to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetDepositVerifier });
-    const transaction = await web3.eth.sendTransaction({ gas: gas, to: gnosisMultisigAddress, from: deployerAddress, data: gnosisSetDepositVerifier });
+    console.log(`The amount of gas used for ${whatLog}: ${gas}`);
```

Comment:
Can we log the correct value of gas sent here too ( *2 ) and for all the logs

Reply:
uuuuups!!!